stories/shared/IconOptions.tsx

```diff
@@ -54,4 +54,4 @@ export const Icons = [
 , "trash"
 , "share"
 , "mail"
-, "lock"]
+, "lock"].sort()
```

Comment:
We can add a comment saying that the default sort order is ascending.

Reply:
Added in next commit